src/site/content/en/blog/bluetooth/index.md

```diff
@@ -0,0 +1,549 @@
+---
+title: Communicating with Bluetooth devices
+subhead: | 
+  The Web Bluetooth API allows websites to communicate with Bluetooth devices.
+authors:
+  - beaufortfrancois
+date: 2015-07-21
+updated: 2020-10-19
+hero: hero.jpg
+thumbnail: thumbnail.jpg
+alt: A Bluetooth chip on a coin
+description: | 
+  The Web Bluetooth API allows websites to communicate with Bluetooth devices.
+tags:
+  - blog # blog is a required tag for the article to show up in the blog.
+  - capabilities
+  - devices
+feedback:
+  - api
+---
+
+What if I told you websites could communicate with nearby Bluetooth devices
+in a secure and privacy-preserving way? This way, heart rate monitors, singing
+lightbulbs, [turtles] could interact directly with a website.
+
+Until now, the ability to interact with bluetooth devices has been possible
+only for native apps. The Web Bluetooth API aims to change this and brings it
+to web browsers as well.
+
+## Before we start
+
+This article assumes you have some basic knowledge of how Bluetooth Low
+Energy (BLE) and the [Generic Attribute Profile] (GATT) work.
+
+Even though the [Web Bluetooth API specification] is not finalized yet, the spec
+authors are actively looking for enthusiastic developers to try out this API and
+give [feedback on the spec] and [feedback on the implementation].
+
+A subset of the Web Bluetooth API is available in Chrome OS, Chrome for Android
+6.0, Mac (Chrome 56) and Windows 10 (Chrome 70). This means you should be able
+to [request] and [connect to] nearby Bluetooth Low Energy devices,
+[read]/[write] Bluetooth characteristics, [receive GATT Notifications], know
+when a [Bluetooth device gets disconnected], and even [read and write to
+Bluetooth descriptors].
+
+For Linux and earlier versions of Windows, enable the
+`#experimental-web-platform-features` flag in `chrome://flags`.
+
+### Available for Origin Trials
+
+In order to get as much feedback as possible from developers using the Web
+Bluetooth API in the field, we've previously added this feature in Chrome 53 as
+an [origin trial] for Chrome OS, Android, and Mac.
+
+The trial has successfully ended in January 2017.
+
+## Security Requirements
+
+To understand the security tradeoffs, I recommend the [Web Bluetooth Security
+Model] post from Jeffrey Yasskin, a software engineer on the Chrome team,
+working on the Web Bluetooth API specification.
+
+### HTTPS Only
+
+Because this experimental API is a powerful new feature added to the Web, it is
+made available only to [secure contexts]. This means you'll need to build with
+TLS in mind.
```

Comment:
```suggestion
[TLS](https://en.wikipedia.org/wiki/Transport_Layer_Security) in mind.
```
I'm not sure if we can expect everyone to know how TLS relates to HTTPS so I think it's safer to provide a link

Reply:
Done.